modules/monitoring-granting-user-permissions-using-the-cli.adoc

```diff
@@ -6,7 +6,12 @@
 [id="granting-user-permissions-using-the-cli_{context}"]
 = Granting user permissions by using the CLI
 
-You can grant users permissions to monitor their own projects, by using the OpenShift CLI (`oc`).
+You can grant users permissions for the `openshift-monitoring` project or their own projects, by using the OpenShift CLI (`oc`).
```

Comment:
hmm `openshift-monitoring` seems off here since we bind either a user-defined project or openshift-user-workload-monitoring. Isn't it redundant since we say elsewhere (`Granting users permission to monitor user-defined projects`) that roles should be bound against openshift-user-workload-monitoring?

Reply:
Oh yes, I tried to make this section so that it also applies for the permissions for core platform monitoring, because we do not have a procedure for those (but as far as I am aware, the procedure should be the same?) But you are right that since this section is nested under UWM, it is a little awkward. Maybe I could raise an issue that we are missing this for core platform monitoring so that I can keep that in mind and add something in the future? And keep this section as it was. We are planning to divide UWM and core platform docs anyway. WDYT?

Reply:
As long as we keep track of the small discrepancy, I'm :+1:

Reply:
Adding a link to the issue that will track this: https://issues.redhat.com/browse/OBSDOCS-1285